core/railing/railing.py

```diff
@@ -22,6 +22,7 @@ def create_railing(bm, faces, prop, normal):
     bmesh.ops.delete(bm, geom=faces, context="FACES")  # delete reference faces
 
 
+@map_new_faces(FaceMap.RAILING_POSTS, skip=FaceMap.RAILING_RAILS)
 def make_corner_posts(bm, edges, prop):
```

Comment:
Note here that the facemap groups for RAILING_POSTS is not yet initialized, it is done later in function `make_fill`.
This function should make a call to `add_facemap_for_groups((FaceMap.RAILING_POSTS)` at the beginning.